lib/room.rb

```diff
@@ -0,0 +1,19 @@
+module HotelBooking
+  class Room
+    attr_reader :number, :cost, :in_block
+    attr_writer :in_block, :cost
+    def initialize(number:, cost: 200, in_block: false)
+      @number = number
+      @cost = cost
+      @in_block = in_block
```

Comment:
The funky thing about putting in_block as an attribute of Room is that it is only in a block for one specific reservation.
When this gets set to true, it will now be marked as true for all reservations that contain this room.
Happy to talk about this in-person if my explanation isn't clear.